test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/ArrayDequeTest.scala

```diff
@@ -0,0 +1,160 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.javalib.util
+
+import org.junit.Test
+import org.junit.Assert._
+
+import scala.collection.JavaConversions._
+
+import java.util.ArrayDeque
+
+class ArrayDequeTest {
+
+  @Test def `should_add_and_remove_head_and_last`(): Unit = {
```

Comment:
I follow this style:
https://github.com/scala-js/scala-js/blob/master/test-suite/shared/src/test/scala/org/scalajs/testsuite/compiler/BooleanTest.scala#L15

please agree on one and let me know ;-)